src/core/constants.mjs

```diff
@@ -79,6 +79,7 @@ export const DEFAULT_CHART_REPO = new Map()
 
 // ------------------- Hedera Account related ---------------------------------------------------------------------------------
 export const OPERATOR_ID = process.env.SOLO_OPERATOR_ID || '0.0.2'
+export const FREEZE_ADMIN_ACCOUNT = '0.0.58'
```

Comment:
allow user override with env var, I noticed that this is actually configurable in Hedera Services.

```suggestion
export const FREEZE_ADMIN_ACCOUNT = process.env.FREEZE_ADMIN_ACCOUNT || '0.0.58'
```